powered-by.html

```diff
@@ -763,6 +763,11 @@
         "logo": "nussknacker.svg",
         "logoBgColor": "#ffffff",
         "description": "Nussknacker is a low-code tool that allows IT teams to hand over decision algorithms to non-technical users. Apache Kafka is Nussknacker's primary input and output interface in streaming use cases - Nussknacker reads events from Kafka, applies decision algorithms and outputs actions to Kafka."
+    }, {
+	"link": "https://www.edenlab.io",
```

Comment:
Seems this does not exist any longer...

Reply:
I've updated the url and the logo in this PR https://github.com/apache/kafka-site/pull/502